tests/jax/test_fused_attn.py

```diff
@@ -754,10 +788,18 @@ def test_backward(
         dtype,
         qkv_layout,
         bias_shape,
+        swa,
     ):
         """
         Test backward with parameterized configs
         """
+        window_size = (-1, -1)
+        if swa:
+            window_size = (s_kv // 10, 0)
+            if s_q > s_kv:
+                pytest.skip(
+                    "seqlen_q > seqlen_kv is not supported with sliding window attention in cuDNN"
+                )
```

Comment:
I noticed that the current [nvte_get_fused_attn_backend](https://github.com/NVIDIA/TransformerEngine/blob/a4e647ed22ee2fc6200a1750189c3a620552c3fb/transformer_engine/common/fused_attn/fused_attn.cpp#L183) only allow `s_q == s_kv`, could you help check if s_q < s_kv unit tests really run?